common/scala/src/main/scala/whisk/core/WhiskConfig.scala

```diff
@@ -264,9 +264,9 @@ object WhiskConfig extends Logging {
     val actionInvokePerMinuteDefaultLimit = "defaultLimits.actions.invokes.perMinute"
     val actionInvokeConcurrentDefaultLimit = "defaultLimits.actions.invokes.concurrent"
     val triggerFirePerMinuteDefaultLimit = "defaultLimits.triggers.fires.perMinute"
-
+    val actionSequenceLimit = "limits.actions.sequence.max"
     val actionInvokePerMinuteLimit = "limits.actions.invokes.perMinute"
     val actionInvokeConcurrentLimit = "limits.actions.invokes.concurrent"
     val triggerFirePerMinuteLimit = "limits.triggers.fires.perMinute"
-
+    val actionSequenceMaxLimit = Map(actionSequenceLimit -> null)
```

Comment:
Where you require the property, add the code above. It doesn't belong in whisk config IMO.